L/libclangex/build_tarballs.jl

```diff
@@ -8,34 +8,40 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "llvm.jl"))
 
 name = "libclangex"
-version = v"0.1.8"
+repo = "https://github.com/Gnimuc/ClangCompiler.jl.git"
+version = v"0.2.0"
 
-llvm_versions = [#=v"11.0.1",=# v"12.0.1", v"13.0.1", v"14.0.5"]
```

Comment:
Doesn't this need to be built for different LLVM versions? Or do you only want to be able to load this on a single Julia version?

Reply:
At the moment I only need LLVM 18/Julia 12, I think I do need to build for different LLVM versions in the future.